pages/common/exa.md

```diff
@@ -26,3 +26,7 @@
 - List files sorted by modification date (oldest first):
 
 `exa --long --sort={{modified}}`
+
+- Long list of files with header, icons, git status:
+
+`exa --long --header --icons --git`
```

Comment:
:thinking: What if we are `--git-ignore` in the example? or we can add a disclaimer in the example description like 
```
(use `--git-ignore` to ...)
```

Reply:
@bl-ue you :+1: which option ?

Reply:
the help says:
```
--git-ignore               ignore files mentioned in '.gitignore'
```
basically you don't see files or folders ignored by git in the output list, e.g. `node_modules/`

Reply:
Maybe we can add a whole new example, too.  But I don't even think it matters that much—this is @ZabejAga's PR, and they can decide which option (if any) they'd like.  @ZabejAga?